index.js

```diff
@@ -125,9 +130,12 @@ program
 
         const prevContributions = await tryAndWait(sequencer, sessionID);
         const receipt = await runCeremony(sequencer, sessionID, entropy, prevContributions);
+        const receiptJson = JSON.stringify(receipt, null, '\t');
 
-        // TODO: Store this receipt?
-        console.log(receipt);
+        fs.writeFile('receipt.json', receiptJson, (err) => {
```

Comment:
How do you think that using this routine as synchonously?
All process could be finished before the writing completed.

Reply:
Wirte file as synchonously 
https://github.com/dsrvlabs/czg-keremony/pull/29/commits/0259b8645162214fd4be6042a919a76527203627